style.css

```diff
@@ -0,0 +1,203 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700&display=swap");
+
+:root {
+  --primary-color: #36b37e;
+  --secondary-color: #172b4d;
+  --primary-font: "Poppins", sans-serif;
+  --secondary-font: "Roboto", sans-serif;
+  --tertiary-font: "Inter", sans-serif;
+  --skills-list-bg: #ebf0ee;
+  --skills-list-color: #3a4a42;
+  --p-text-color: #42526e;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  overflow-x: hidden;
+}
+
+.main-container {
+  display: flex;
+  position: relative;
+  align-content: space-between;
+  background: #fff;
+  width: 375px;
+  height: 718px;
+  top: 1px;
+}
+
+.main-container .topBar-nav {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+}
+
+.main-container .toolBar-nav .toolBar-nav {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 44px;
+  bottom: 0;
+  height: 48px;
+  width: 375px;
+  border-radius: 0;
+}
+
+/* where the Logo Placed */
+
+.main-container .toolBar-nav .logo-setting {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: absolute;
+  width: 100px;
+  height: 40px;
+  left: 12px;
+  top: calc(50% - 40px / 2);
+  border-radius: 0;
+  padding: 10px 12px 10px 12px;
+}
+
```

Comment:
As part of the project requirements, kindly ensure that your design matches that of the actual design in terms of dimensions like margins, paddings, fonts,etc. I can see there is a big white space to the right side of the design as shown below. Kindly consider updating it to match the actual design :+1: 


### Your design 


![Screenshot from 2023-03-23 23-17-07](https://user-images.githubusercontent.com/65233860/227387026-2560ea26-19b5-4ab1-86c9-bac052410ecf.png)



### Actual design 


![Screenshot from 2023-03-23 23-33-41](https://user-images.githubusercontent.com/65233860/227387005-5f811feb-fe54-4dce-8c1b-9bbc91b54e44.png)